forms-web-app/src/views/check-answers.njk

```diff
@@ -111,8 +111,8 @@
               text: "Planning application number"
             },
             value: {
-              text: ""
-            },
+            text: appeal['application-number']
```

Comment:
I don't think the indentation is correct

Reply:
cheers, fixed